sdks/go/test/load/group_by_key/group_by_key.go

```diff
@@ -0,0 +1,106 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is GroupByKey load test with Synthetic Source. Besides of the standard
+// input options there are additional options:
+// * fanout (optional) - number of GBK operations to run in parallel
+// * iterations (optional) - number of reiterations over per-key-grouped
+// values to perform
+// input_options - options for Synthetic Sources.
+
+// Example test run:
+
+// go run sdks/go/test/load/group_by_key_test/group_by_key.go  \
```

Comment:
It's good to copy-and-paste such examples to check if everything's alright. It seems something's wrong here:
`panic: Could not unmarshal SourceConfig: invalid character '\\' looking for beginning of object key string`

To be honest, I have mixed feelings about such examples. They can be useful in shell scripts, where there are plenty of positional arguments. But in this case, all arguments are named, and only one of them is required. What's more, those examples can easily become outdated, because it's easy to miss them while editing the file. That happened once to Python load tests.

If you decide to keep it, I guess you will have to remove backslashes to make it work.